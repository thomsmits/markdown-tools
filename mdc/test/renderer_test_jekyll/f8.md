# Chapter
## Slide 6.1

Die Formel ist \[ f(x) = x^2 \] aber hier haben wir einen x^2 exponenten CO_2
