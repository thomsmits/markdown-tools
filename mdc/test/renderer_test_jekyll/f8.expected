---
layout: page
parent: Section 3
title: Chapter
nav_order: 0
---
<h1 class="fw-700 text-purple-300">Chapter</h1>
## Slide 6.1

<p>Die Formel ist \\( f(x) = x^2 \\) aber hier haben wir einen x<sup>2</sup> exponenten CO<sub>2</sub>
</p>



